x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/package_policy_input_var_field.tsx

```diff
@@ -420,6 +420,7 @@ function SecretInputField({
           iconType="refresh"
           iconSide="left"
           size="xs"
+          data-test-subj={`button-replace${lowercaseTitle}`}
```

Comment:
I think it's better to reuse `fieldTestSelector` instead of `lowercaseTitle` here